ui/pages/settings/contact-list-tab/edit-contact/edit-contact.component.js

```diff
@@ -74,13 +75,21 @@ export default class EditContact extends PureComponent {
           className="settings-page__header address-book__header--edit"
           paddingLeft={6}
           paddingRight={6}
+          width={BlockSize.Full}
+          alignItems={AlignItems.center}
         >
-          <Box display={Display.Flex} alignItems={AlignItems.center}>
+          <Box
+            display={Display.Flex}
+            alignItems={AlignItems.center}
+            style={{ overflow: 'hidden' }}
+            paddingRight={2}
+          >
             <AvatarAccount size={AvatarAccountSize.Lg} address={address} />
             <Text
               className="address-book__header__name"
               variant={TextVariant.bodyLgMedium}
               marginInlineStart={4}
+              style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}
```

Comment:
Can you use the ellipsis prop here? https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-text--docs#ellipsis

<img width="1507" alt="Screenshot 2024-06-06 at 12 31 14 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/73d713ce-b924-4e2e-829a-d9f0f2e19383">

```suggestion
              ellipsis
```